external/kafka-0-10/src/main/scala/org/apache/spark/streaming/kafka010/package.scala

```diff
@@ -17,7 +17,52 @@
 
 package org.apache.spark.streaming
 
+import org.apache.spark.internal.config.ConfigBuilder
+
 /**
  * Spark Integration for Kafka 0.10
  */
-package object kafka010 //scalastyle:ignore
+package object kafka010 { //scalastyle:ignore
```

Comment:
Last thing: I think you can take off the scalastyle:ignore?

Reply:
> Last thing: I think you can take off the scalastyle:ignore?

The author of this object use this comment. I don't know the reason. If you think this comment 
is useless, I will remove it.

Reply:
As I know the reason is the digits in the object name, like here:

https://github.com/apache/spark/blob/1d95dea30788b9f64c5e304d908b85936aafb238/external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/package.scala#L21-L23

Reply:
Oh OK, I thought it was because it's empty. Leave it then.